.buildkite/pipeline.yml

```diff
@@ -0,0 +1,30 @@
+plugin-node-cache: &plugin-node-cache
+  cache_node: true
+
+plugins: &plugins
+  - ssh://git@github.com/goodeggs/goodeggs-core-buildkite-plugin#v0.0.18: *plugin-node-cache
+
+steps:
+  - label: 'Install'
+    command: NODE_ENV=development yarn install --frozen-lockfile
+    plugins: *plugins
+
+  - wait
+
+  - label: 'All Tests'
+    command: yarn test
+    plugins: *plugins
+
+  - label: ':rocket: Deploy'
```

Comment:
this deploy step is for an application, whereas this is a node module.  here are a couple examples of node modules that are already on buildkite to use as an example:

```
meaty~/Projects % ag 'yarn publish' */.buildkite/pipeline.yml
goodeggs-authentication-tokens/.buildkite/pipeline.yml
41:      - yarn publish

goodeggs-kodif-domain-events-adapter-contracts/.buildkite/pipeline.yml
25:      - yarn publish

goodeggs-orders-service-contracts/.buildkite/pipeline.yml
25:      - yarn publish

goodeggs-product-feed-contracts/.buildkite/pipeline.yml
25:      - yarn publish

goodeggs-tsconfig/.buildkite/pipeline.yml
```

Reply:
Nice, thank you!